blog/2022/02/10years.md

```diff
@@ -688,3 +688,8 @@ My first encounter with the Julia language happened in fall of 2014. At the time
 As a systems biologist, I form part of a very interdisciplinary field. I work in teams of biologists, mathematicians, computer scientists and more. Leveraging expertise of multiple domains is key for many hard problems in the field. One key aspect I consider when choosing my programming language is how it enables collaborations. A focus of Julia’s language design is abstraction and its implied modularity and composability. The effect that this design emphasis has on my day to day work of collaborating with other researchers is hard to describe in only a few words. Simple, yet clear syntax as well as elegant technical features, such as multiple dispatch, solve the expression problem and make it convenient to reuse, adapt and extend existing code of collaborators. Additionally, exploiting Julia’s rich package ecosystems is made easy, too.
 Second, Julia is fast. In the biological sciences, the amount of data at our fingertips is growing rapidly. Software that scales well is of great importance. In order to built sophisticated models that learn large biological systems using big data, I need a level of performance that languages such as R, Python or MATLAB cannot deliver. Two-language approaches, where code is translated into a faster language as a second stage of a project, are popular but also inherently inefficient and can be a source of mistakes and inaccuracy. Julia solves this two-language problem. Due to its easy syntax, Julia gives me a convenient environment for prototyping and algorithm design as well as production ready, high performance code for data-heavy work.
 Third, I use Julia because it is free, open source and because of its community. This motivates a constant peer reviewing process and a democratic approach to software development, publicly accessible for anyone to join on GitHub. The community guidelines and an active and welcoming user base bring together not only great coders but also kind and ambitious people — the ideal place for great things to happen.
+
+## Leandro Martínez [(@lmiq)](https://github.com/m3g)
```

Comment:
Just double-checking: the text of the link is your user handle `@lmiq`, but the link points to https://github.com/m3g.  Is that intentional?

Reply:
Yes, it is, the useful software I write goes to the group  (m3g) account.